mvp-spec/partials-updater/src/files.ts

```diff
@@ -2,10 +2,20 @@ import RootPath from "app-root-path";
 import fs from "fs";
 import path from "path";
 
+const BinDir = path.join(RootPath.path, "node_modules", ".bin");
```

Comment:
For another commit: variables should usually be camelCase.
https://standardjs.com/rules.html

Reply:
Done